topics/single-cell/tutorials/scatac-standard-processing-snapatac2/tutorial.md

```diff
@@ -0,0 +1,1017 @@
+>    - {% icon param-file %} *"Annotated data matrix"*: `Anndata 5k PBMC gene_matrix magic UMAP`
+>    - *"Function to manipulate the object"*: `Add new annotation(s) for observations or variables`
+>        - *"What to annotate"*: `Observations (obs)`
+>        - {% icon param-file %} *"Table with new annotations"*: `Cell type annotation`(output of **Replace column** {% icon tool %})
+> 9. Rename the generated file to `Anndata 5k PBMC gene_matrix magic cell_type` or add the tag {% icon galaxy-tags %} `cell_type` to the dataset
+>
+> 10. {% tool [Plot with Scanpy](toolshed.g2.bx.psu.edu/repos/iuc/scanpy_plot/scanpy_plot/1.9.6+galaxy3) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: `Anndata 5k PBMC gene_matrix magic cell_type` (output of **Manipulate AnnData** {% icon tool %})
+>    - *"Method used for plotting"*: `Embeddings: Scatter plot in UMAP basis, using 'pl.umap'`
+>        - *"Keys for annotations of observations/cells or variables/genes"*: `cell_type`
+>        - {% icon param-toggle %} *"Show edges?"*: `No`
+>        - In *"Plot attributes"*
+>           - *"Location of legend"*: `on data`
+>           - {% icon param-toggle %} *"Draw a frame around the scatter plot?"*: `Yes`
+> 11. {% icon galaxy-eye %} Inspect the `.png` output
+>
+>  ![UMAP annotated cell types]({% link topics/single-cell/images/scatac-standard-snapatac2/umap_cell-types.png %})
+>
+>    > <question-title></question-title>
+>    > 1. Are clusters with the same assigned cell type located close to each other?
+>    > 2. Are the [expected cell type percentages of {PBMCs}](https://www.akadeum.com/blog/what-are-pbmc-human-pbmc-cells/) visible in the annotated plot?
+>    >    
+>    >    Cell Type | Expected Percentage
+>    >    --- | ---
+>    >    CD3+ T cells | 50-70%
+>    >    CD4+ T cells | 25-45%
+>    >    CD8+ T cells | 10-25%
+>    >    B cells | 5-15%
+>    >    NK cells | <5%
+>    >    Monocytes | 10-30%
+>    >    Dendritic cells | <2%
+>    > 
+>    > > <solution-title></solution-title>
+>    > >
+>    > > 1. Yes. B-cells are far away from NK and T cells. Only the myeloid lineage of monocytes and dendritic cells are located close to each other. This might be due to the common progenitor cell lineage and thus a similar chromatin profile.  
+>    > > 2. Yes. The most common cell types are T cells, followed by Monocytes and B cells. NK cells and Dendritic cells only make up a small percentage of PBMCs.  
+>    > {: .solution}
+>    >
+>    {: .question}
+>
+{: .hands_on}
+
+
+# Conclusion
+{% icon congratulations %} Well done, you’ve made it to the end! You might want to consult your results with this [control history](https://singlecell.usegalaxy.eu/u/timonschlegel/h/single-cell-atac-seq-standard-processing-with-snapatac2), or check out the [full workflow](https://usegalaxy.eu/u/timonschlegel/w/workflow---standard-processing-of-10x-single-cell-atac-seq-data-with-snapatac2) for this tutorial.
```

Comment:
```suggestion
{% icon congratulations %} Well done, you’ve made it to the end! You might want to consult your results with this [control history](https://usegalaxy.eu/u/timonschlegel/h/single-cell-atac-seq-standard-processing-with-snapatac2), or check out the [full workflow](https://usegalaxy.eu/u/timonschlegel/w/workflow---standard-processing-of-10x-single-cell-atac-seq-data-with-snapatac2) for this tutorial.
```
I think you should stay consistent, either use the subdomain in all links or not. The workflow is not using it so maybe better to use the plain domain?

Reply:
Good idea!